feelpp/feel/feelmodels/modeloutputs.cpp

```diff
@@ -52,16 +52,25 @@ ModelOutput::ModelOutput( std::string name, pt::ptree const& p, worldcomm_ptr_t
     if( auto markers = p.get_child_optional("markers") )
         M_markers.setPTree(*markers);
     else
-        LOG(WARNING) << "Output " << M_name << " does not have any range\n";
+        LOG(INFO) << "Output " << M_name << " does not have any range\n";
 
     if( auto t = p.get_optional<int>("topodim") )
```

Comment:
it seems we are using boost property_tree, why is that ? I thought we moved away, no?

Reply:
Not yet, I think @vincentchabannes is doing it in the branch feature/toolboxes